src/redux/books/bookSlice.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booksArr: [],
+  categoriesArr: [],
+};
+
+// Includes an array of books
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+    },
+    removeBook: () => {
+    },
+  },
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Zven94/Bookstore/assets/77069011/5bee11e3-67df-4a9c-93e6-cfbeba0326a8)

Please consider implementing both, the `addBook` and `removeBook` reducers! 🤓 🔧